### ISSUE TITLE
Organise committee meetings

### ISSUE THREAD

**Issue by href** [python-summit/SPSA#3]
I'm not quite sure yet how often we should organise committee meetings, how we should hold them and who signs the protocols.

We discussed before that we want to do this online, so google hangouts or chats would be great and I'm all in favour of those.

So the question is how often.

I'm also unclear on the protocol. It looks like they are usually signed by the meeting chair as well as the person doing the protocol. Since we do those online, how is this supposed to work? If we just did this in a chat, could we possibly just print the chat log?

**Comment by The-Compiler** [comment]
> I'm also unclear on the protocol. It looks like they are usually signed by the meeting chair as well as the person doing the protocol. Since we do those online, how is this supposed to work? If we just did this in a chat, could we possibly just print the chat log?

FWIW in the Chaos Computer Club Zurich we simply publish the logs on the mailinglist, and if after $time nobody objects, they get added to the website.

**Comment by dbrgn** [comment]
> who signs the protocols.

I don't think we need to sign them at all. I'd simply post them to the mailing list.

> We discussed before that we want to do this online, so google hangouts or chats would be great and I'm all in favour of those.

I agree. We could use a service like https://appear.in/ that does not require any plugins.
